src/eradiate/pipelines/_assemble.py

```diff
@@ -81,23 +89,19 @@ def eval_illumination_spectrum(
             # Local helper function to help with illumination spectrum evaluation
 
             spectrum: Spectrum = getattr(illumination, field_name)
+            wavelengths = to_quantity(x.w)
 
             if eradiate.mode().is_mono:
-                # Very important: sort spectral coordinate
-                wavelengths = np.sort(measure.spectral_cfg.wavelengths)
+                # Very important: sort spectral coordinate  # TODO: is it?
```

Comment:
I think this is not necessary because it is garuanteed by the current SRF-compatible `Spectrum` subclasses:
* In `InterpolatedSpectrum`, `wavelengths` validator will raise if values are not monotonically increasing
* In `MultiDeltaSpectrum`, `wavelengths` converter will sort them